examples/list/list.icon.tsx

```diff
@@ -0,0 +1,61 @@
+import { type CodeData } from '~/components/code-demo';
+import { List, ListItem } from '~/src';
+import { HiCheckCircle } from 'react-icons/hi';
+
+const code = `
+'use client';
+
+import { List } from 'flowbite-react';
+
```

Comment:
missing `import { HiCheckCircle } from 'react-icons/hi';`

Reply:
Thanks @SutuSebastian I will fix it now